examples/bert/run_pretraining.py

```diff
@@ -142,7 +142,7 @@ def get_config(self):
         return config
 
 
-class MaskedLM(keras.layers.Layer):
+class MaskedLMHead(keras.layers.Layer):
```

Comment:
Curious - why are we renaming it to MaskedLMHead?

Reply:
I just wanted to get in the habit of naming all these layer that attach to transformers ouputs for loss function "heads." So you have MaskedLMHead and ClassificationHead in this file. I think the uniformity in naming will help make the example clearer.

Not that this is head as in a "screwdriver head," a different piece you attach to a standard tool. I think it's common usage when discussing these models.